lib/jobs/job.go

```diff
@@ -0,0 +1,79 @@
+package jobs
+
+import (
+	"fmt"
+	"strings"
+
+	yaml "gopkg.in/yaml.v2"
+
+	"github.com/kosyfrances/rundeck-zabbix/lib"
+	"github.com/kosyfrances/rundeck-zabbix/lib/zabbix"
+)
+
+type nodeFilters struct {
+	Filter string `yaml:"filter"`
+}
+
+type commands struct {
+	Exec string `yaml:"exec"`
+}
+
+type sequence struct {
+	Commands []commands `yaml:"commands"`
+}
+
+type job struct {
+	Name        string      `yaml:"name"`
+	Description string      `yaml:"description"`
+	NodeFilters nodeFilters `yaml:"nodefilters"`
+	Sequence    sequence    `yaml:"sequence"`
+}
+
+/*
+Make generates Rundeck job model document in Yaml format
+with triggers' information from Zabbix
+
+The file output similar to:
+
+- name: test-job
+  description: Just a random test
+  nodefilters:
+    filter: 'dummy-host'
+  sequence:
+    commands:
+    - exec: ""
+*/
+func Make(results zabbix.TriggerResults, file, prefix string) error {
```

Comment:
`file` argument is a string here. 
I suggest to name it as `fileName` or `filePath` depending on which it is to make it clearer that it isn't a file object